draws/models.py

```
@@ -1,10 +1,112 @@
+        }
+
+# Model for creating a new draw
+class DrawCreate(BaseModel):
+    category_id: str
+    status: str = "pending_open" # Default status for a newly created scheduled draw
+    scheduled_open_time: datetime
+    scheduled_close_time: datetime
+    # created_at and updated_at will be set in DB layer
+
+    @classmethod
+    def from_category(cls, category: 'lottery_categories.models.LotteryCategory', start_time: Optional[datetime] = None):
```

Flake8 (7.2.0)

[error] undefined name 'lottery_categories'

(F821)